Makefile.am

```diff
@@ -25,7 +25,11 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-SUBDIRS = u2f-server src tests gtk-doc
+SUBDIRS = u2f-server src gtk-doc
+
+if ENABLE_TESTS
+	SUBDIRS+=tests
```

Comment:
This doesn't work for me.
Automake directives need *not* to be indented (sigh...).

Could you please change that?